.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,11 @@
+# Contributing to Mobify Code Style
+
+## Bugs, Feature Requests
+Submit an [issue](https://github.com/mobify/mobify-code-style/issues).
+
+## Pull Requests
+
+1. Make a new branch from `develop`.
```

Comment:
This is a point for the broader repo, but is there any reason we can't use the "continuous deployment" workflow for this repo? In that case we would branch of `master`...